pkg/snapshot/snapshotprovisioner.go

```diff
@@ -74,6 +84,35 @@ func (p *snapshotProvisioner) snapshotRestore(
 	return pvSrc, labels, err
 }
 
+func (p *snapshotProvisioner) isSnapshotAllowed(
+	snapshot crdv1.VolumeSnapshot,
+	namespace string,
+) bool {
+	allowedNamespaces, ok := snapshot.Metadata.Annotations[storkSnapshotRestoreNamespacesAnnotation]
+	if !ok {
+		return false
+	}
+
+	csvReader := csv.NewReader(strings.NewReader(allowedNamespaces))
+	namespaces, err := csvReader.ReadAll()
+	if err != nil {
+		log.Errorf("Error parsing allowed namespaces: %v", allowedNamespaces)
+		return false
+	}
+
+	for _, namespaceRegEx := range namespaces[0] {
```

Comment:
this will panic if `allowedNamespaces = ""` 

`csvReader.ReadAll` does not return err in this case.

Reply:
Good catch. Fixed